packages/espressocash_app/lib/features/incoming_split_key_payments/src/bl/iskp_service.dart

```diff
@@ -69,7 +69,7 @@ class ISKPService {
         cluster: apiCluster,
       );
 
-      final response = await _client.receivePayment(dto);
+      final response = await _client.receivePaymentEc(dto);
```

Comment:
We should support both versions of links for some time.

Reply:
@ookami-kb updated code to support both. should have v3 links in app